src/function/statistics/correlation.js

```diff
@@ -0,0 +1,60 @@
+import { factory } from '../../utils/factory.js'
+
+const name = 'corr'
+const dependencies = ['typed']
+
+export const createCorrelation = /* #__PURE__ */ factory(name, dependencies, ({ typed }) => {
+  /**
+   * Compute the colleration of a two list with values.
+   *
+   * Syntax:
+   *
+   *     math.corr(A, B)
+   *
+   * Examples:
+   *
+   *     math.corr([1, 2, 3, 4, 5], [4, 5, 6, 7, 8])     // returns 1
+   *     math.corr([1, 2.2, 3, 4.8, 5], [4, 5.3, 6.6, 7, 8])     // returns 0.9569941688503644
+   *
+   */
+  return typed(name, {
+    // correlation(xArray, yArray)
+    'Array, Array': function (xArray, yArray) {
```

Comment:
Can you also implement a signature `Matrix, Matrix`? Like:

```js
'Matrix, Matrix': function (x, y) {
  return matrix(_corr(x.toArray(), y.toArray())
}
```

Reply:
I have pushed the necessary changes, including implementation for signature Matrix, Matrix.

Reply:
Thanks, looks good